CONTRIBUTING.md

```diff
@@ -39,12 +37,9 @@ Feel free to check out the `#discussion`/`#development` channels on our [Slack](
 
 ## Developing
 
-**Note:** Versions `< 5.1.10` can't be built.
+*Node*: Check that Node is [installed](https://nodejs.org/en/download/) with version 6 and up. You can check this with `node -v`.
```

Comment:
Let's put 10 here. We guarantee that Babel runs on Node.js 6, but I'm not sure about our devtools.
Only `make bootstrap` / `make test` are guaranteed to run on Node.js 6 because we use them on CI.